templates/api.mustache

```diff
@@ -77,7 +77,7 @@ func (a *{{{classname}}}Service) {{{nickname}}}(ctx context.Context{{#hasParams}
 	)
 
 	// create path and map variables
-	localVarPath := a.client.cfg.BasePath + "{{{path}}}"{{#pathParams}}
+	localVarPath := "{{{path}}}"{{#pathParams}}
```

Comment:
What do you think about changing this to use path.Join (also adding the import to the template)?
```suggestion
	localVarPath := path.Join(a.client.cfg.BasePath, "{{{path}}}"{{#pathParams}})
```

This way I think it wouldn't be necessary to also override the `configuration.mustache` template.

Reply:
@cezarsa path.Join does not work as expected. I just moved the changes to api.mustache only. PTAL